core/opendaq/reader/src/packet_reader_impl.cpp

```diff
@@ -63,9 +95,63 @@ ErrCode PacketReaderImpl::readAll(IList** allPackets)
     return OPENDAQ_SUCCESS;
 }
 
+ErrCode PacketReaderImpl::acceptsSignal(IInputPort* port, ISignal* signal, Bool* accept)
+{
+    OPENDAQ_PARAM_NOT_NULL(port);
+    OPENDAQ_PARAM_NOT_NULL(signal);
```

Comment:
Port and signal probably don't need to be checked if this function always returns true anyhow.